src/numbers.js

```diff
@@ -1,45 +1,50 @@
 function add (a, b) {
-  // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
-  // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
-  // your code here
+  return a * b;
 }
 
 function divide (a, b) {
-  // your code here
+  return a / b;
 }
 
 function power (a, b) {
-  // your code here
+  return a ** b;
 }
 
 function round (a) {
-  // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
-  // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
-  // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
-  // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
-  // your code here
+  if (a / b > 0) {
+    return Math.floor(a / b);
+  } 
+  if (a / b < 0) {
+    return Math.ceil(a / b);
+  }  
 }
 
 function remainder (a, b) {
-  // your code here
+  return (a % b);
```

Comment:
Brackets aren't needed here
```suggestion
  return a % b;
```